alns/weights/__init__.py

```diff
@@ -1,3 +1,8 @@
-from .SegmentedWeights import SegmentedWeights
-from .SimpleWeights import SimpleWeights
-from .WeightScheme import WeightScheme
+import warnings
+
+from alns.select import RouletteWheel as SimpleWeights
+from alns.select import SegmentedRouletteWheel as SegmentedWeights
+
+message = "alns.weights has been deprecated in favour of alns.select. \
+See https://github.com/N-Wouda/ALNS/issues/74 for details."
```

Comment:
Fairly minor, but maybe refactor?
```python
message = "alns.weights has been deprecated in favour of alns.select. " \
          "See https://github.com/N-Wouda/ALNS/issues/74 for details."
```

So everything's on the same indentation level.

Reply:
Black has it's own way of doing this
```python
message = (
    "alns.weights has been deprecated in favour of alns.select."
    "See https://github.com/N-Wouda/ALNS/issues/74 for details."
)
```

Reply:
Yeah that works too. Black can sort that out. 

Very nit, but it's missing a space after the first sentence/before the second. Now the message will read
```
alns.weights has been deprecated in favour of alns.select.See https://github.com/N-Wouda/ALNS/issues/74 for details.
```